docs/user-documentation/iiif.md

```diff
@@ -0,0 +1,40 @@
+# IIIF (International Image Interoperability Framework)
+
+[IIIF](https://iiif.io/) is a set of specifications that provides interoperability of image-based collections across platforms. What this means for a repository platform like Islandora at a general level is that image-based objects such as (still) images and paged content can be managed by Islandora but viewed in external applications, and that Islandora can bring in image-based content from elsewhere to supplement locally managed content. If this intrigues you, see the section "Looking under the hood (and beyond)" below.
+
+At a practial level, because Islandora supports several of the IIIF specifications, we can:
+
+- Zoom, pan, and rotate images within OpenSeadragon
+   - Islandora uses an IIIF-compliant image server (by default, [Cantaloupe](https://cantaloupe-project.github.io/)) that utilizes the [IIIF Image API](https://iiif.io/api/image/2.1/). This capablity is similar to what Islandora 7.x users experience when they view a Large Image.
+- Display thumbnails for all pages of a book or newspaper issue within image viewers
+   - IIIF-compliant image viewers such as OpenSeadragon or Mirador can display a "collection" of images such as all the pages of a book or newspaper issue using the [IIIF Presentation API](https://iiif.io/api/presentation/2.1/). For example, here is a screenshot of OpenSeadragon rendering all the pages of a book:
+   
+   ![OpenSeadragon rendering book pages](../assets/osd_collection_mode.png)
+
+## Using IIIF in Islandora 8
+
+Implementation of the IIIF Presentation API is new in Islandora 8, and using it is as simple as configuring a Context (in fact, you don't even need to configure it, the Context, Openseadragon Block, already exists by default).
+
+To use this Context, all that is required is for your book or newspaper (or other paged content) to be given a model of "Paged Content" or "Publication Issue". Then, in the Openseadragon Block Context, make sure the term used in the "Node has term" condition (you can register more than one term there). Now, when you view a paged content Islndora object, you will see thumbnails of all of its child pages (assuming you have added some child pages to the object) in the OpenSeadragon viewer as illustrated above. 
+
+You can change how the paged content thumbnails are arranged in the OpenSeadragon viewport by doing the following:
```

Comment:
Hi Mark - running through the steps below, I can't seem to successfully change "collection layout" from vertical to horizontal. Going through the steps, the thumbnails are still rendering as vertical in the viewer.

Reply:
I'm afraid I'm having some problems with OpenSeadragon at the moment. I wonder if someone else like @manez , @seth-shaw-unlv , or @dannylamb  can try to replicate what you're seeing.

Reply:
I am not able to reproduce @cderven's issue. Changing the selection, saving the settings, and refreshing the _Paged Content_ node's page is allowing me to toggle back and forth with no issues.

Perhaps you need to clear your cache or do a hard refresh on your browser?

Reply:
Are there any error messages appearing in your Drupal error logs (`/admin/reports/dblog`) or in the browser's JS console?

Reply:
Thanks @seth-shaw-unlv, this was a browser issue and I can verify @mjordan's description now of adjusting these settings. I'll mark this as resolved.